.eslintrc

```diff
@@ -0,0 +1,52 @@
+{
+  "env": {
+    "amd": true,
+    "es6": true,
+    "jasmine": true,
+    "node": true,
+    "mocha": true,
+    "browser": true,
+    "builtin": true
+  },
+  "rules": {
+    "camelcase": 0,
+    "comma-dangle": 0,
+    "complexity": [2, 6],
+    "consistent-return": 0,
+    "curly": [2, "all"],
+    "dot-notation": 0,
+    "eqeqeq": [2, "allow-null"],
+    "global-strict": [0, "never"],
+    "globals": {
+      "define": true,
+      "Promise": true
+    },
+    "handle-callback-err": 0,
+    "key-spacing": 0,
+    "new-cap": 0,
+    "no-cond-assign": [2, "except-parens"],
+    "no-debugger": 2,
+    "no-empty": 0,
+    "no-eval": 2,
+    "no-irregular-whitespace": 2,
+    "no-loop-func": 0,
+    "no-multi-spaces": 0,
+    "no-new": 2,
+    "no-script-url": 2,
+    "no-sequences": 2,
+    "no-shadow": 0,
+    "no-spaced-func": 0,
+    "no-undef": 2,
+    "no-underscore-dangle": 0,
+    "no-unused-vars": [1, {vars: all, args: none}],
```

Comment:
Not sure if these are strictly valid JSON, or if they need quotes.
Alternatively, we could (should?) convert this file from JSON to YAML because YAML is super awesome.